modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -116,6 +123,22 @@ publish: Internal
 By setting this option, you create an internal Ingress Controller and a private load balancer.
 endif::restricted[]
 
+ifdef::localzone[]
+. Edit the `install-config.yaml` file to provide the subnets for the availability zones that your VPC uses:
++
+[source,yaml]
+----
+compute:
+platform:
+  aws:
+    subnets: <1>
+    - subnet-1
+    - subnet-2
+    - subnet-3
+----
+<1> Add the `subnets` section and specify the `PublicSubnetIds` from the output of the CloudFormation template for the VPC.
```

Comment:
This is supposed to be the public subnets, right?

Reply:
Sorry, you must include the private subnet here too. Take a look in my suggestion below.